plugins/ingest-attachment/src/main/plugin-metadata/plugin-security.policy

```diff
@@ -43,4 +43,5 @@ grant {
   permission java.lang.RuntimePermission "accessDeclaredMembers";
   // PDFBox checks for the existence of this class
   permission java.lang.RuntimePermission "accessClassInPackage.sun.java2d.cmm.kcms";
+  permission java.io.FilePermission "/System/Library/Fonts/-", "read";
```

Comment:
This is the issue with using existing running gradle daemon, if we kill the running gradle daemon and re-run the test we dont see this error.